win32ss/user/user32/controls/appswitch.c

```diff
@@ -225,9 +225,9 @@ BOOL IsAltTabWindow(HWND hwnd)
     if (!IsWindowVisible(hwnd))
         return FALSE;
 
-    // must not be WS_EX_TOOLWINDOW
+    // must not be WS_EX_TOOLWINDOW nor WS_EX_NOACTIVATE
     ExStyle = GetWindowLong(hwnd, GWL_EXSTYLE);
-    if (ExStyle & WS_EX_TOOLWINDOW)
```

Comment:
There are other similar checks ("if window not visible or is toolbox...") in this code; should some of them be complemented with the no-activate check as well?

Reply:
I think nope.